tests/baselines/reference/asyncMethodWithSuperConflict_es6.js

```diff
@@ -2,14 +2,19 @@
 class A {
     x() {
     }
+    y() {
+    }
 }
 
 class B extends A {
     // async method with only call/get on 'super' does not require a binding
     async simple() {
         const _super = null;
+        const _superProps = null;
```

Comment:
Is this intended to verify the file-level unique name for the helper? If so this needs to be renamed to `_superIndex` to match the updated name.

Reply:
Good catch, done.